tests/test_github.py

```diff
@@ -206,3 +206,74 @@ def test_pr_list(self, mock_request):
         self.assertEqual(prccf.deletions, 21)
         self.assertEqual(prccf.changes, 103 + 21)
         self.assertEqual(prccf.patch, "@@ -132,7 +132,7 @@ module Test @@ -1000,7 +1000,7 @@ module Test")
+
+    @patch('prSHARK.backends.github.Github._send_request', side_effect=mock_return)
+    def test_update_without_changes(self, mock_request):
+
+        cfg = Namespace(tracking_url='https://localhost/repos/smartshark/test/pulls')
+
+        project = Project.objects.get(name='test')
+
+        vcs_system = VCSSystem(project_id=project.id, url='https://github.com/octocat/Hello-World.git',
+                               repository_type='git', collection_date=datetime.datetime.now())
+        vcs_system.save()
+
+        c = Commit(vcs_system_ids=[vcs_system.id], revision_hash='6dcb09b5b57875f334f61aebed695e2e4193db5e')
+        c.save()
+
+        mc = Commit(vcs_system_ids=[vcs_system.id], revision_hash='e5bd3914e2e596debea16f433f57875b5b90bcd6')
+        mc.save()
+
+        gp1 = Github(cfg, project)
+        gp1.run()
+        gp1 = Github(cfg, project)
+        gp1.run()
+        pr = PullRequest.objects.get(external_id='1347')
+        prr = PullRequestReview.objects.filter(pull_request_id=pr.id)
+        prrc = PullRequestReviewComment.objects.filter(external_id='10')
+        prc = PullRequestComment.objects.filter(pull_request_id=pr.id)
+        pre = PullRequestEvent.objects.filter(pull_request_id=pr.id)
+        prccf = PullRequestFile.objects.filter(pull_request_id=pr.id)
+
+        self.assertEqual(len(pr.pull_request_system_ids), 2)
+        self.assertEqual(len(prr), 1)
+        self.assertEqual(len(prrc), 1)
+        self.assertEqual(len(prc), 1)
+        self.assertEqual(len(pre), 1)
+        self.assertEqual(len(prccf), 1)
+
+    @patch('prSHARK.backends.github.Github._send_request', side_effect=mock_return)
+    def test_update_with_changes(self, mock_request):
```

Comment:
Could you write a short comment what you expect to happen, what you test here?
Is it just one additional pull request or was a comment added?